lab1.py

```diff
@@ -0,0 +1,56 @@
+# Task1
+
+from typing import AsyncGenerator
+
+
+print("Enter first number ")
+a = int(input())
+print("Enter second number ")
+b = int(input())
+
+print("Prime numbers")
+for i in range(a, b + 1):
+ 
+    if (i == 1):
+        continue
+    flag = 1    
+    for j in range(2, i // 2 + 1):
+        if (i % j == 0):
+            flag = 0
+            break
+    if (flag == 1):
+        print(i,  " ")
+
+# Task2
+
+
+def show_student(name,gpa):
```

Comment:
As required you need to add here `(name, GPA=0)`
So it will take the default value in the definition if the user didn't pass a value.